templates/pimcore5/files/.platform.app.yaml

```diff
@@ -0,0 +1,100 @@
+# This file describes an application. You can have multiple applications
+# in the same project.
+#
+# See https://docs.platform.sh/user_guide/reference/platform-app-yaml.html
+
+# The name of this app. Must be unique within a project.
+
+
+# The runtime the application uses.
+name: pimcore
+type: php:7.3
+
+
+# The relationships of the application with services or other applications.
+#
+# The left-hand side is the name of the relationship as it will be exposed
+# to the application in the PLATFORM_RELATIONSHIPS variable. The right-hand
+# side is in the form `<service name>:<endpoint name>`.
+relationships:
+  database: "pimcore_mysqldb:pimcore_admin"   
+  redis: 'rediscache:redis'
+
+# Add additional PHP extensions.
+build:
+  flavor: none
+
+runtime:
+  extensions:
+    - redis
+    - iconv
+    - dom
+    - simplexml
+    - exif
+    - opcache
+    - imagick
+    - fileinfo
+
+variables:
+  env:
+    PIMCORE_INSTALL_ADMIN_USERNAME: "admin"
+    PIMCORE_INSTALL_ADMIN_PASSWORD: "admin"
+  php:
+      memory_limit: 512M
```

Comment:
Dear god, what an obscenely wasteful application.  Is that necessary?

Reply:
https://pimcore.com/docs/5.x/Development_Documentation/Installation_and_Upgrade/System_Requirements.html they suggest more or equal to 128MB, 512MB be an overkill I'll bring it down to 128MB as suggested or 256MB, I have no experience with php about the memory usage.

Reply:
Most well-behaved modern systems shouldn't need 512 MB, ever.  A well-behaved system will probably only ever need 20-30 MB, but will periodically spike over 100.  A 128 MB limit is probably fine.  We can easily bump it to 256 in the future if needs be.

Reply:
Indeed, setting the memory ton 128MB then.